controller/shelve.js

```diff
@@ -7,7 +7,7 @@ const sessions = require('./sessions');
 
 module.exports = function () {
   console.log(clc.blue('Beginning shelve process!'));
-  projects.action = () => 'shelve';
+  projects.action = 'shelve';
```

Comment:
was this being invoked somewhere?

Reply:
I put it back. It was changed for a type error caused by [this code](https://github.com/OperationSpark/opspark/pull/77/commits/abaf0d25b4fcc274a62488b91d4c1a111bf0a21e#diff-b79016cc4e946d93321a22954145a985306746d64d3d614d3dbcac51805bcf8dL39). It's fixed now

Reply:
word